recipes/awscli2/meta.yaml

```diff
@@ -0,0 +1,143 @@
+{% set name = "awscli" %}
+{% set version = "2.13.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+# v2 is not on PyPI https://github.com/aws/aws-cli/issues/6785
+source:
+  url: https://github.com/aws/aws-cli/archive/{{ version }}.tar.gz
+  sha256: 523646f7b86b4beef74c0f0e9a2a178ea1d40a21b7da0e6852db17d76044b07c
+
+build:
+  number: 0
+  # Build uses pep517 with a custom backend
+  # console_scripts entrypoints aren't used, so this can't be noarch
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv  # [not unix]
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv && mkdir -p $PREFIX/share/bash-completions/completions && echo complete -C aws_completer aws >$PREFIX/share/bash-completions/completions/aws  # [unix]
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    # The autocomplete index step is run on the build host
+    - colorama >=0.2.5,<0.4.7                # [build_platform != target_platform]
+    - docutils >=0.10,<0.20                  # [build_platform != target_platform]
+    - cryptography >=3.3.2,<=40.0.2          # [build_platform != target_platform]
+    - ruamel.yaml >=0.15.0,<=0.17.21         # [build_platform != target_platform]
+    - ruamel.yaml.clib >=0.2.0,<=0.2.7       # [build_platform != target_platform]
+    - prompt_toolkit >=3.0.24,<3.0.39        # [build_platform != target_platform]
+    - distro >=1.5.0,<1.9.0                  # [build_platform != target_platform]
+    - awscrt >=0.16.4,<=0.17.0               # [build_platform != target_platform]
+    - python-dateutil >=2.1,<3.0.0           # [build_platform != target_platform]
+    - jmespath >=0.7.1,<1.1.0                # [build_platform != target_platform]
+    - urllib3 >=1.25.4,<1.27                 # [build_platform != target_platform]
+
+  host:
+    - python
+    - flit-core >=3.7.1,<3.8.1
+    - pip
+    # The build process generates the autocomplete index (scripts/gen-ac-index)
+    # which requires importing awscli modules, so we need to install all runtime
+    # dependencies too. YAML anchors don't support merging lists so we have to
+    # duplicate everything :-(
+    - colorama >=0.2.5,<0.4.7
+    - docutils >=0.10,<0.20
+    - cryptography >=3.3.2,<40.0.2
+    - ruamel.yaml >=0.15.0,<=0.17.21
+    - ruamel.yaml.clib >=0.2.0,<=0.2.7
+    - prompt-toolkit >=3.0.24,<3.0.39
+    - distro >=1.5.0,<1.9.0
+    - awscrt >=0.16.4,<=0.16.16
+    - python-dateutil >=2.1,<3.0.0
+    - jmespath >=0.7.1,<1.1.0
+    - urllib3 >=1.25.4,<1.27
+    # See https://github.com/aws/aws-cli/issues/7942
+    - pyopenssl <23.2
```

Comment:
was this an addition of yours @xhochy? what should we keep an eye on to see whether this can be dropped in the future?